src/components/Nav.jsx

```diff
@@ -33,9 +31,9 @@ const Nav = () => {
             )
           } else {
           return(
-            <li key={slug}>
-              <p><strong>{slug}</strong></p>
-            </li>
+            <Link to={`/articles/?topic=${slug}`}>
```

Comment:
There's a warning in the console for missing keys. I think it might be because there is no key for those that are rendered from the else block.